Project.toml

```diff
@@ -21,6 +21,7 @@ NCDatasets = "85f8d34a-cbdd-5861-8df4-14fed0d494ab"
 Oceananigans = "9e8cae18-63c1-5223-a75c-80ca9d6e9a09"
 OffsetArrays = "6fe1bfb0-de20-5000-8ca7-80f57d26f881"
 OrthogonalSphericalShellGrids = "c2be9673-fb75-4747-82dc-aa2bb9f4aed0"
+PrecompileTools = "aea7be01-6a6a-4083-8856-8a6e6704d82a"
```

Comment:
You'll need to add a compat bound as well 😉 "1" could be a decent initial guess

Reply:
ah ya